### PR TITLE
Bump CentOS os 1905_1

### PR DESCRIPTION
**Pull request by rmohr**

### CODE REVIEW

File: cluster-provision/README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 * `kubevirtci/cli`: `sha256:1dd015dea4f12e6dcb8e31be3eeb677fed96f290ef4a4892a33c43d666053536`
 * `kubevirtci/gocli`: `sha256:caf1c3f63dc5f3137795084e492a48c802a7ef2e7f7fbe1d67e5cff684d376a3`
 * `kubevirtci/base`: `sha256:034de1a154409d87498050ccc281d398ce1a0fed32efdbd66d2041a99a46b322`
-* `kubevirtci/centos:1804_02`: `sha256:70653d952edfb8002ab8efe9581d01960ccf21bb965a9b4de4775c8fbceaab39`
+* `kubevirtci/centos:1905_01`: `sha256:f2b204a3fabf71494c3cf7a145dae88e63f068a84ca10149b0aec523045dc2c1`
```
Comment (fromanirh) [inline cluster-provision/README.md]: Looks like 1905 is a pre-release build of upcoming centos 8. I think we should upgrade to 1902 instead. Rationale:
https://app.vagrantup.com/centos/boxes/7/versions/1902.01

**Comment by fromanirh** [comment]
> just curious, is there a special reason for the update?

Yes. The reason is a common-templates change is failing CI because of out of date kernel: https://github.com/kubevirt/common-templates/pull/70#issuecomment-503983165

**Comment by fromanirh** [comment]
Hey, I'm not clear what's the next step here. If something's on me, please repeat! :)

File: cluster-provision/README.md (hunk 1 above)
Comment (slintes) [reply cluster-provision/README.md]: I think the documention was outdated, 1905 is announces in the meanwhile :)
https://blog.centos.org/2019/07/updated-centos-vagrant-images-available-v1905-01/
https://app.vagrantup.com/centos/boxes/7/versions/1905.1

Comment (fromanirh) [reply cluster-provision/README.md]: @slintes yep, good point. Good for us!

**Comment by slintes** [comment]
rebased and updated shasums
@fromanirh fyi

/lgtm
/approve

**Comment by fromanirh** [comment]
/lgtm

thanks @slintes !

**Pull request merged**